src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -110,16 +110,16 @@ interface Account where
     do
       debit this arg
 
-  nonconsuming choice Transfer : ContractId Transferable.I
+  nonconsuming choice CreditAndDebit : ContractId Lockable.I
     -- ^ Transfers a `Transferable` to this `Account`.
```

Comment:
```suggestion
    -- ^ Used in the context of a `Transfer` to debit an existing `Transferable` ands credit it to the `Account`.
```

Feel free to apply if you find this descriptive, otherwise feel free to ignore the comment